src/components/application_manager/test/commands/mobile/put_file_test.cc

```diff
@@ -293,9 +292,8 @@ TEST_F(PutFileRequestTest, Run_AddFile_SUCCESS) {
   EXPECT_CALL(*mock_app_, folder_name()).WillOnce(Return(kAppFolder));
 
   const uint32_t avaliable_space = 2u;
-  EXPECT_CALL(*mock_app_, GetAvailableDiskSpace())
-      .WillOnce(Return(avaliable_space))
-      .WillOnce(Return(avaliable_space));
+  EXPECT_CALL(*mock_app_, GetAvailableDiskSpace()).Times(3)
```

Comment:
@LitvinenkoIra please avoid hadrcoding number of times this method will be called. Use WillRepeatedly

Reply:
@LuxoftAKutsan Done 7017728df8a6b226ee81a6e6f35268b03cbc41b9